sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_devices_daily_events_v1/query.sql

```diff
@@ -0,0 +1,118 @@
+WITH fxa_events AS (
+  SELECT
+    `timestamp`,
+    user_id,
+    IF(service IS NULL AND event_type = 'fxa_activity - cert_signed', 'sync', service) AS service,
+    device_id,
+    os_name,
+    flow_id,
+    event_type,
+    country,
+    `language`,
+    entrypoint,
+    utm_term,
+    utm_medium,
+    utm_source,
+    utm_campaign,
+    utm_content,
+    ua_version,
+    ua_browser,
+  FROM
+    `moz-fx-data-shared-prod.firefox_accounts.fxa_content_auth_oauth_events` -- TODO: this will need updated to fxa_all_events once unified
+  WHERE
+    DATE(`timestamp`)
+    BETWEEN DATE_SUB(@submission_date, INTERVAL 1 DAY)
+    AND @submission_date
+    AND event_type NOT IN ( --
+      'fxa_email - bounced',
```

Comment:
I'd add a comment here that this exclusion list logic is a carry over of logic from the users_services daily. We are keeping it for now to maintain consistency as we do not have the documentation to determine if we should remove.